huaweicloud/lb_v2_shared.go

```diff
@@ -92,6 +93,46 @@ func waitForLBV2LoadBalancer(networkingClient *golangsdk.ServiceClient, id strin
 	return nil
 }
 
+// for v2 api
+func waitForLBV2LoadBalancer_v2(networkingClient *golangsdk.ServiceClient, id string, target string, pending []string, timeout time.Duration) error {
+	log.Printf("[DEBUG] Waiting for loadbalancer %s to become %s.", id, target)
```

Comment:
the log content shouldn't contain a period.

Reply:
fixed